apps/namadillo/src/hooks/useShouldInvalidateShieldedContext.ts

```diff
@@ -0,0 +1,37 @@
+import { chainParametersAtom } from "atoms/chain/atoms";
+import {
+  indexerHeartbeatAtom,
+  lastInvalidateShieldedContextAtom,
+} from "atoms/settings/atoms";
+import { useAtomValue } from "jotai";
+import { useEffect } from "react";
+import semverSatisfies from "semver/functions/satisfies";
+import { useInvalidateShieldedContext } from "./useInvalidateShieldedContext";
+
+// change this value to the minimun indexer version that demands to clear the context
+const minIndexerVersion = ">=9.9.0";
```

Comment:
This version is a higher number on purpose, to avoid invalidating the current context from the current user without providing a real user value.

When we have the final indexer version, with masp updated, we only need to update this number